docs/ruff.toml

```diff
@@ -0,0 +1 @@
+line-length=78
```

Comment:
This seems to be the longest length that doesn't scroll horizontally on the website. I tested by inspect element then adding characters until the box scrolled on Firefox.